cms/djangoapps/contentstore/tests/test_admin.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests that verify that the admin view loads.
+
+This is not inside a django app because it is a global property of the system.
+"""
+import ddt
+from django.test import TestCase
+from django.urls import reverse
+from edx_toggles.toggles.testutils import override_waffle_flag
+
+from openedx.core.djangoapps.user_authn.config.waffle import ADMIN_AUTH_REDIRECT_TO_LMS
+
+
+@ddt.ddt
+class TestAdminView(TestCase):
+    """
+    Tests of the admin view.
+    """
+    @override_waffle_flag(ADMIN_AUTH_REDIRECT_TO_LMS, True)
+    @ddt.data(('/admin/', 3), ('/admin/login', 3), (reverse('admin:login'), 2))
+    @ddt.unpack
+    def test_admin_login_redirect(self, admin_url, index):
```

Comment:
Looks like the `index` parameter can now be removed (and unpack as well, I suppose.)

Reply:
Updated test parameters.